src/views/data/data-view-filters-container.js

```diff
@@ -233,33 +207,174 @@ export class DataViewFiltersContainer extends LitElement {
               >
               </multiselect-combo-box>
             `}
+        <br />
         ${this.filterCategoriesDataLoading
           ? html`
               <span>Loading...</span>
             `
           : html`
               <multiselect-combo-box
-                Label="Filter by collection:"
+                Label="Exclude collections:"
                 id="filter-collection"
                 item-label-path="categoryname"
                 style="display: ${this.shouldShowFilterDropdown()
                   ? 'inline-flex'
                   : 'none'}"
                 @selected-items-changed="${this
-                  .handleCategoriesComboBoxChanged}"
+                  .handleCategoriesExcludeComboBoxChanged}"
                 .items="${this.filterCategoriesData}"
                 item-value-path="category"
               >
               </multiselect-combo-box>
             `}
+        ${this.filterFilesDataLoading
+          ? html`
+              <span>Loading...</span>
+            `
+          : html`
+              <multiselect-combo-box
+                Label="Exclude files:"
+                id="filter-filename"
+                style="display: ${
+                  this.shouldShowFilterDropdown() ? 'inline-flex' : 'none'
+                }"
+                item-label-path="categoryname"
+                @selected-items-changed="${
+                  this.handleFilesExcludeComboBoxChanged
+                }"
+                .items="${this.filterFilesData}"
+                item-value-path="filename"
+              >
+              </multiselect-combo-box>
+            </div>`}
+      </div>
+    `;
+  }
+  createFilterParameters() {
```

Comment:
there's a lot of HTML in this file. Maybe this should be a separate component in a separate file? Typically, if a function only returns HTML, it's a component, and thus should be named like `FilterParameters` and be defined outside of this component class. 
This helps ensure it will be reusable and not use the parent's component state directly.

Reply:
I agree but for now I will just put a TODO on it.